HW6/VZinkov/HW_6_3.py

```diff
@@ -0,0 +1,13 @@
+def count(str1):
```

Comment:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result